robot.py

```diff
@@ -61,9 +61,7 @@ def teleopPeriodic(self):
         wpilib.SmartDashboard.putNumber("centreVelocity", inner_throttle)
 
         if self.joystick_left.getRawButtonPressed(11):
-            self.loading_piston.set(wpilib.DoubleSolenoid.Value.kForward)
-        if self.joystick_left.getRawButtonPressed(12):
-            self.loading_piston.set(wpilib.DoubleSolenoid.Value.kReverse)
+            self.loading_piston.startPulse()
```

Comment:
Robot.py should probably not initiate this directly, this should be done from within the component